src/pages/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useState } from "react";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import { CopyIcon, GithubIcon } from "../components/Icons";
+
+import { ThemeConfig } from "@docusaurus/preset-classic";
+import Features from "../components/Features";
+
+function copyToClipboard(text: string) {
+  if (navigator.clipboard) {
+    navigator.clipboard.writeText(text);
+    return;
+  }
+
+  var textArea = document.createElement("textarea");
+  textArea.value = text;
+
+  textArea.style.top = "0";
+  textArea.style.left = "0";
+  textArea.style.position = "fixed";
+
+  document.body.appendChild(textArea);
+  textArea.focus();
+  textArea.select();
+
+  try {
+    document.execCommand("copy");
+  } catch (err) {}
+
+  document.body.removeChild(textArea);
+}
+
+export default function Home() {
+  const {
+    siteConfig: { themeConfig, tagline },
+  } = useDocusaurusContext();
+  const headerImg = (themeConfig as ThemeConfig).navbar!.logo!;
+
+  const [copied, setCopied] = useState(false);
+
+  const copyCommandHandler = useCallback(() => {
+    if (copied) return;
+
+    copyToClipboard(
+      "dart pub global activate stacked_cli\nstacked create app awesome_app"
+    );
+    setCopied(true);
+
+    setTimeout(() => setCopied(false), 3000);
+  }, []);
+
+  return (
+    <>
+      <header className="sticky top-0 flex items-center justify-between px-6 py-4 shadow bg-white/20 backdrop-blur-md">
+        <div className="flex items-center pointer-events-none select-none">
+          <img className="h-8" src={headerImg.src} alt={headerImg.alt} />
+          <strong className="pl-4 text-xl">Stacked</strong>
+        </div>
+
+        <a
+          href="/"
+          className="flex items-center px-4 py-2 text-lg text-black transition-all bg-gray-100 rounded-full outline-none ring-transparent focus:ring-purple-500 ring hover:no-underline hover:text-black hover:bg-gray-200"
+        >
+          <GithubIcon className="mr-4" />
+          Github
+        </a>
+      </header>
+      <main className="px-6 py-1 min-h-[70vh] flex flex-col items-center justify-center">
+        <h1 className="text-5xl font-black text-center md:text-7xl">
+          The
+          <span className="text-transparent bg-clip-text bg-gradient-to-br from-indigo-800 to-purple-400">
+            scalable
```

Comment:
Change the main title to "The [Flutter Framework] for Production" The text in `[]` should be the new purple. For production should be on the second line.